src/main/java/org/spongepowered/api/effect/particle/ParticleEffect.java

```diff
@@ -26,12 +26,16 @@
 
 import com.flowpowered.math.vector.Vector3d;
 import org.spongepowered.api.Sponge;
-import org.spongepowered.api.util.ResettableBuilder;
+import org.spongepowered.api.data.DataSerializable;
+import org.spongepowered.api.data.persistence.DataBuilder;
+
+import java.util.Map;
+import java.util.Optional;
 
 /**
  * Represents a particle effect that can be send to the Minecraft client.
  */
-public interface ParticleEffect {
+public interface ParticleEffect extends DataSerializable {
```

Comment:
@gabizou You may want to make particle effects configurable in a plugin, that is why I made it serializable.